doc/controls/SafeArea.md

```diff
@@ -176,8 +175,248 @@ Bottom TabBar:
+                <Grid Padding="50,0">
+                    <!--  0: Logo, 1: Spacer, 2: FormPanel  -->
+                    <Grid.RowDefinitions>
+                        <RowDefinition Height="Auto" />
+                        <RowDefinition Height="40" />
+                        <RowDefinition Height="*" />
+                    </Grid.RowDefinitions>
+                    ...
+    +            </Grid>
+    +        </ScrollViewer>
+        </Grid>
+    </Page>
+    ```
+
+    </td>
+    </tr>
+    </table>
+
+2. If you do not want to have the whole page scrollable, you could wrap only the FormPanel `StackPanel` within a `SafeArea` instead of the root `Grid`. This may not always have the best visual effect as can be seen in the demonstration below
+
+    <table>
+    <tr>
+        <th>Page</th>
+        <th>XAML</th>
+    </tr>
+    <tr>
+    <td><img src="../assets/safearea-login-scroll-stackpanel.gif" width="400px"/> </td>
+        <td>
+
+    ```diff
+    <Page xmlns:utu="using:Uno.Toolkit.UI">
+        ...
+    +    <utu:SafeArea Insets="SoftInput"
+    +                  Grid.Row="2">
+    +        <StackPanel Spacing="20">
+    -                    Grid.Row="2">
+                <TextBlock Text="Welcome to Uno!"
+                            HorizontalAlignment="Center"
+                            FontSize="35" />
+                <PersonPicture ProfilePicture="ms-appx:///Assets/profile.png" />
+                <TextBox x:Name="TextBox"
+                            PlaceholderText="Username" />
+                <PasswordBox x:Name="PasswordBox"
+                                PlaceholderText="Password" />
+                <StackPanel Orientation="Horizontal"
+                            Spacing="4">
+                    <CheckBox Padding="0"
+                                MinWidth="0" />
+                    <TextBlock Text="Remember me"
+                                VerticalAlignment="Center" />
+                </StackPanel>
+                <Button Content="Login"
+                        x:Name="LoginBtn"
+                        VerticalAlignment="Top"
+                        HorizontalAlignment="Stretch"
+                        Margin="0,30" />
+            </StackPanel>
+    +    </utu:SafeArea>
+        ...
+    </Page>
+    ```
+
+    </td>
+    </tr>
+    </table>
+
 ## Notes
 
 - `SafeArea` is able to adapt to views that are only _partially_ obscured by applying the minimum amount of `Padding`/`Margin` needed until the content is fully inside the visible bounds.
 - When a control already has a non-zero `Padding`/`Margin`, `SafeArea` takes those values into consideration when calculating the minimum amount of pixels needed for the view to be within the safe area.
-- `SafeArea` on WinAppSDK/Desktop does not have any effect. It is present to allow for same-XAML across platforms.
\ No newline at end of file
+- `SafeArea` on WinAppSDK/Desktop does not have any effect. It is present to allow for same-XAML across platforms.
```

Comment:
```suggestion
- `SafeArea` on WinAppSDK/Desktop currently does not have any effect. It is present to allow for same XAML across platforms.
```

Reply:
this doesnt work on uwp on windows and winui on windows? or just the latter?

Reply:
Doesn't work on WinUI Windows